### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by Ian-MacLeod**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "simplejwt-extensions"
+version = "0.1.0"
+description = ""
+authors = ["Ian MacLeod <ian.a.macleod@gmail.com>"]
```
Comment (markrawls) [inline pyproject.toml]: Bequest, Inc

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
```
Comment (markrawls) [inline pyproject.toml]: There should be a key in here that tells PyPi where the README is at.

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+name = "simplejwt-extensions"
+version = "0.1.0"
+description = ""
+authors = ["Bequest, Inc. <oss@willing.com>"]
+readme = "README.md"
```
Comment (markrawls) [inline pyproject.toml]: The README is not in Markdown, it's in RST.

Comment (Ian-MacLeod) [reply pyproject.toml]: Whoops, updated

**Pull request merged**